### PR TITLE
Make the existing ReinterpretFramebuffers/ShaderColorBitmask path work for Split/Second

### LINKED ISSUE #13957: Split-Second Missing HUD for 3rd person view
I tried OpenGL and Vulkan the HUD for 3rd person view in Split-Second is broken in latest builds of PPSSPP

Here's a comparison:

v1.10.3-352-gd90630c26 and newer: 
Screenshot_2021-01-22-12-49-33-907_org ppsspp ppsspp

v1.10.3-348-gdd79d33f1 and older: 
Screenshot_2021-01-22-12-40-31-719_org ppsspp ppsspp

Here's a GE dump: 
[ULUS10513_0001.ppdmp.zip](https://github.com/hrydgard/ppsspp/files/5853665/ULUS10513_0001.ppdmp.zip)

### PR DESCRIPTION
**Pull request by hrydgard**
Fixes #13957.

It took writing and debugging #15500 for me to understand what the issue with the old path was..

Much simpler alternative to #15500, or we could merge both but disable Split/Second for this one. Needs some benchmarks I guess...

### CODE REVIEW

**Comment by hrydgard** [comment]
I think we should definitely merge this one actually, and only then consider merging #15500.

File: GPU/Common/DrawEngineCommon.cpp [hunk 1]
```diff
@@ -501,7 +501,7 @@ bool DrawEngineCommon::ApplyFramebufferRead(bool *fboTexNeedsBind) {
 		lastFrameBlit = gpuStats.numFlips;
 	}
 	++blitsThisFrame;
-	if (blitsThisFrame > MAX_REASONABLE_BLITS_PER_FRAME * 2) {
+	if (blitsThisFrame > MAX_REASONABLE_BLITS_PER_FRAME * 4) {
```
Comment (unknownbrackets) [inline GPU/Common/DrawEngineCommon.cpp:504]: Should we just increase MAX_REASONABLE_BLITS_PER_FRAME?  It seems weird to multiply the constant, when this is its only use...

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

**Comment by Gamemulatorer** [comment]
Performance on vulkan become slow unlike to opengl it's the same 🤔

### vulkan
Screenshot_2022-04-25-15-20-59-527_org ppsspp ppsspp

### opengl
Screenshot_2022-04-25-15-22-55-074_org ppsspp ppsspp

**Comment by hrydgard** [comment]
@Gamemulatorer Check how performance is now :)

File: GPU/Common/DrawEngineCommon.cpp (hunk 1 above)
Comment (hrydgard) [reply GPU/Common/DrawEngineCommon.cpp:504]: I'm just gonna remove the whole thing, see #15515

**Comment by Gamemulatorer** [comment]
> @Gamemulatorer Check how performance is now :)

Ok will I try :)

**Comment by Gamemulatorer** [comment]
What's happening :(
Screenshot_2022-05-01-21-27-48-128_org ppsspp ppsspp

**Comment by Gamemulatorer** [comment]
Both happen with ogl and vk

**Comment by Panderner** [comment]
@Gamemulatorer latest build?

**Comment by hrydgard** [comment]
@Gamemulatorer  Maybe your game ID is missing from compat.ini ? What is it?

**Comment by unknownbrackets** [comment]
Is that black thing new since that recent merge, though?  Even if the game ID was missing, it would seem bad if the change made it worse in that way.

-[Unknown]

**Comment by hrydgard** [comment]
I can't figure out how the change would cause that, works here on several devices. Maybe you have an older compat.ini that neither enables the old nor the new workaround, though in that case, it shouldn't have worked for you previously either...

**Comment by Panderner** [comment]
Works fine for me on PPSSPP v1.12.3-1138-g36bc7a3e6
Screenshot (30)

But it has bugs for software rendering for this game:
Screenshot (29)

**Comment by Gamemulatorer** [comment]
Nevermind I forget that I have a compat.ini file in system folder.
But it's ok now using Vulkan.
Screenshot_2022-05-02-18-18-35-988_org ppsspp ppsspp

**Comment by hrydgard** [comment]
Great, thanks for reporting!

I have plans for some other changes that should speed the game up further, but they're a bit invasive and will need to wait for after the upcoming release.

**Pull request merged**